frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -43,6 +40,12 @@ const store = mockStore({
   facility: { id: "1", name: "Facility 1" },
 });
 
+const testResults = {
```

Comment:
Defining the mock's shape here rather than in-file since we're testing both component-level and GQL-level functionality in the file